apidoc/Titanium/UI/UI.yml

```diff
@@ -174,6 +174,16 @@ description:  |
 extends: Titanium.Module
 since: "0.4"
 
+events:
+  - name: userInterfaceStyle
+    summary: Fired when the `userInterfaceStyle` changes.
```

Comment:
Shouldn't it be lowercase (`userinterfacestyle`)?

Reply:
I used the camelCase name in the code. but it may be more consistent to use all lowercase?

Reply:
https://jira.appcelerator.org/browse/TIMOB-10291

Reply:
Man, you are really digging up some old tickets/references! 👍 ok, changing to all lowercase